util/check_characters.py

```diff
@@ -18,12 +18,14 @@
 import csv
 import os
 import sys
+import unicodedata
 
 def main():
     parser = argparse.ArgumentParser()
 
     parser.add_argument("-csv", "--csv-files", help="Str. Filenames as a comma separated list", required=True)
     parser.add_argument("-alpha", "--alphabet-format", help="Bool. Print in format for alphabet.txt", action="store_true")
+    parser.add_argument("-unicode", "--unicode-variants", help="Bool. Check unicode consistency (use with --alphabet-format)", type=bool, default=True)
```

Comment:
nit: This still has the counter-intuitive behavior that to disable it you need to do `-unicode ''`, any string of length > 0 as a value will set it to True, as that's what `bool(string)` does in Python.